src/pkg/caendr/caendr/services/cloud/analytics.py

```diff
@@ -37,6 +45,7 @@ def get_weekly_visits(SERVICE):
       'reportRequests': [{
         'viewId': '117392266',
         'dateRanges': [{'startDate': '2015-01-01', 'endDate': datetime.now().date().isoformat()}],
```

Comment:
not part of the PR, but consider creating a constant for `2015-01-01` that offers semantic meaning (aligned with the existing comment).